### ISSUE TITLE
[Two Column Layout] Impossible to select text in title and left slide

### ISSUE THREAD

**Issue by Xartok** [sfeir-open-source/sfeir-school-theme#51]
I created these slides with two-column-layout :

```
<!-- .slide: class="two-column-layout" -->

# Title

##--##

<!-- .slide -->

Left Slide

##--##

Right Slide

##==##
```

This is transformed into this html (simplified version) when going on this slide in presentation mode :

```html
<section class="stack two-column-layout present">
    <section class="past">
        Title
    </section>
    <section class="past">
        Left Slide
    </section>
    <section class="present">
        Right Slide
    </section>
</section>
```

The issue is that the past class apply a `pointer-events: none` that prevent selection, so selection is enabled only on the right slide. It would be better if all these sections were detected as `present`.

**Comment by jefBinomed** [comment]
linked #29

**Comment by jefBinomed** [comment]
Fix avec [#ff5dba5](https://github.com/sfeir-open-source/sfeir-school-theme/commit/ff5dba54967d55c6c3b08e1e04e5ef5ed71fd395)
